src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Devices/Network.vb

```diff
@@ -0,0 +1,31 @@
+' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Option Explicit On
+Option Strict On
+
+Namespace Microsoft.VisualBasic.Devices
+
+    '''**************************************************************************
+    ''';Network
+    ''' <summary>
+    '''  An object that allows easy access to some simple network properties and functionality.
+    ''' </summary>
+    ''' <remarks></remarks>
+    Public Class Network
```

Comment:
Will a future PR be adding the code for this type, and others, in this PR?

Reply:
Yes, the missing members will be added in subsequent PRs.